samples/sdl/main.c

```diff
@@ -173,9 +173,6 @@ static const struct {
   "\0\0\0\0\0\0\0\0\0\0\0",
 };
 
-//Screen dimension constants
-const extern int SCREEN_WIDTH;
```

Comment:
Has already been fixed in upstream in [this commit](https://github.com/XboxDev/nxdk/commit/c17f6223621c754b9e486e21a923dfd84d0b4839).

(valid for all three samples)